lib/main/main_common.dart

```diff
@@ -8,6 +8,7 @@ import '../presentation/app.dart';
 import 'app_flavor.dart';
 
 void mainCommon(AppFlavor appFlavor) {
+  debugPrint('launching....mode.... ${appFlavor.appFlavorType}');
```

Comment:
I would like to present proper capitalisation and punctuation to the rest of MonstarLab members who will use this. Please change to : `Launching Mode : ${appFlavor.appFlavorType}`.